app/styles/partials/_students_index.scss

```diff
@@ -0,0 +1,47 @@
+#wrapper {
+    padding-left: 0;
+}
+
+#page-wrapper {
+    width: 100%;
+    padding: 0 0 5%;
+    //background-color: #fff;
+    margin-top: 1px;
+    //border-left: 1px solid black;
+    min-height: 100%;
+}
+
+.well {
+    overflow-x: auto;
+}
+
+.active-overflow {
+    overflow: auto;
+}
+
+.search-label {
+    min-width: 76px;
+}
+
+button.btn.pageFormButton,
```

Comment:
mmm, I'm not sure these styles (and below ones) should be here. If they are useful, they should be placed at `commons.scss`. Please check this.